lib/SILOptimizer/Mandatory/DIMemoryUseCollector.h

```diff
@@ -130,6 +118,10 @@ class DIMemoryObjectInfo {
   /// Return true if this is 'self' in any kind of initializer.
   bool isAnyInitSelf() const { return !MemoryInst->isVar(); }
 
+  /// Return uninitialized value of 'self' if current memory object
+  /// is located in an initializer (of any kind).
+  SingleValueInstruction *findAnyInitSelfValue() const;
```

Comment:
Alternatively I could call it - `findUninitializedSelfValue()` or something like that...

Reply:
yes I think `findUninitializedSelfValue` would be a better name